profile/config-update.go

```diff
@@ -123,7 +123,8 @@ func updateGlobalConfigProfile(ctx context.Context, task *modules.Task) error {
 		// Add module warning to inform user.
 		module.Warning(
 			globalConfigProfileErrorID,
-			fmt.Sprintf("Failed to process global settings: %s", err),
+			"Internal Settings Failure",
+			fmt.Sprintf("The app settings layering system failed to process the global settings. This means that some global settings might not be applied correctly. You can try restarting the Portmaster to resolve this problem. Refer to the error for more details: %s", err),
 		)
```

Comment:
As with the "filter list system", I would not talk about "app settings layering system". That's way to complex and our users will feel completely overwhelmed by this message. I'd remove the first sentence completely and just say: "Some global settings might not be applied correctly. You can try restarting the Portmaster to resolve this problem. Error: %s"